search_username/search_username/settings.py

```diff
@@ -0,0 +1,101 @@
+"""
+Django settings for project project.
+
+Generated by 'django-admin startproject' using Django 2.1.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '24v*hmmgg0@5^q0k&3lf3wp_z2#(m(b80tf&k#k^zbij@(o5^6'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
+
+# Application definition
+
+INSTALLED_APPS = ['search_username', 'django.contrib.admin', 'django.contrib.auth',
```

Comment:
It looks very strange.
Please fix these apps as it was because your formatting is very bad.

Reply:
done